HelpSource/Classes/List.schelp

```diff
@@ -100,6 +100,35 @@ Inserts the strong::item:: at the beginning of the List.
 
 method::insert
 Inserts the strong::item:: into the contents of the List at the indicated strong::index::.
+code::
+~alist = List[10, 20, 30, 40] // -> List[10, 20, 30, 40]
+~alist.insert(0, 1)           // -> List[1, 10, 20, 30, 40]
+~alist                        // -> List[1, 10, 20, 30, 40]
+::
+
+method::boundedInsert
+Same as link::#-insert::, but it maintains the size of the receiver, and the values for index greater than the size of the link::Classes/List:: will be clipped to the last index.
+code::
+~alist = List[10, 20, 30, 40] // -> List[10, 20, 30, 40]
+~alist.boundedInsert(0, 1)    // -> List[1, 10, 20, 30]
+~alist                        // -> List[1, 10, 20, 30]
+
+~alist = List[10, 20, 30, 40] // -> List[10, 20, 30, 40]
+~alist.boundedInsert(-2, 1)   // List[1, 10, 20, 30]
+~alist                        // List[1, 10, 20, 30]
+
+~alist = List[10, 20, 30, 40] // -> List[10, 20, 30, 40]
+~alist.boundedInsert(2, 1)    // -> List[10, 20, 1, 30]
+~alist                        // -> List[10, 20, 1, 30]
+
+~alist = List[10, 20, 30, 40] // -> List[10, 20, 30, 40]
+~alist.boundedInsert(3, 1)    // -> List[10, 20, 30, 1]
+~alist                        // -> List[10, 20, 30, 1]
+
+~alist = List[10, 20, 30, 40] // -> List[10, 20, 30, 40]
+~alist.boundedInsert(10, 1)   // -> List[10, 20, 30, 1]
+~alist                        // -> List[10, 20, 30, 1]
+::
```

Comment:
Do we need to repeat examples here? Or could we just link to Classes/ArrayedCollection#-boundedInsert ?